Microsoft.Toolkit.Uwp.UI.Controls/CameraPreview/CameraPreview.xaml

```diff
@@ -0,0 +1,38 @@
+﻿<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:Microsoft.Toolkit.Uwp.UI.Controls">
+    <Style x:Name="MessageStyle" TargetType="TextBlock">
+        <Setter Property="FontSize" Value="14"/>
+        <Setter Property="FontWeight" Value="Bold"/>
+        <Setter Property="Foreground" Value="Red"/>
+    </Style>
+    <Style TargetType="local:CameraPreview" >
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="local:CameraPreview">
+                    <Grid Background="{TemplateBinding Background}">
+                        <Grid.RowDefinitions>
+                            <RowDefinition Height="Auto"></RowDefinition>
+                            <RowDefinition Height="Auto"></RowDefinition>
+                        </Grid.RowDefinitions>
+                        <ComboBox x:Name="FrameSourceGroupCombo" Header="Frame Source Group" HorizontalAlignment="Left" Width="Auto">
```

Comment:
Instead of a ComboBox, what if we just have a simple toggle button on top of the MediaPlayerElement to toggle between sources? We should then have a bool dependancy property to enable and disable the button.

If the user wants to provide a ComboBox, can we expose the list of sources in a read only list?